### DISCUSSION TITLE
osu! crashing when you start a beatmap

### DISCUSSION THREAD

**Discussion by 0xdeadbeer** [ppy/osu#16281] [Q&A] [answered]
This is a problem that affects the newest commit apparently, yesterday whenever I opened up osu! lazer and started a beatmap everything worked fine. This morning I woke up only to pull the resource once again and start playing some beatmaps and.. osu! just gave up on me, meaning whenever I start a beatmap it will not wait 3 seconds before crashing.. I'm not sure if I'm the only one having this problem though. 
[runtime.log](https://github.com/ppy/osu/files/7792528/runtime.log)

**Comment by peppy** [comment, marked as answer]
Fixed via https://github.com/ppy/osu/pull/16282.

**Reply by taep96** [reply]
I am still having this issue since version [2022.306.0](https://github.com/ppy/osu/releases/tag/2022.306.0) iirc. Currently on [2022.314.0](https://github.com/ppy/osu/releases/tag/2022.314.0)
[runtime.log](https://github.com/ppy/osu/files/8247145/runtime.log)

**Reply by bdach** [reply]
@taep96 there is no evidence in the log you attached that you are *actually* having the same issue. the log cuts out midway through.

please attach a complete log, or if it always cuts out suddenly, check dmesg or similar for kernel level issues.

**Reply by taep96** [reply]
Sorry, didn't see how old the issue was. After crashing and reading logs a couple of times again, turns out that nothing gets written to logs after. Log folder right after crash:

[logs.zip](https://github.com/ppy/osu/files/8247758/logs.zip)

**Reply by bdach** [reply]
if there is nothing in the log that resembles an error message then i cannot help you because i am not clairvoyant. please check system level logs for any traces of a crash. i believe it's likely that you will find that it be specific to your linux setup or hardware in some respect.

**Reply by taep96** [reply]
I tried running it directly from terminal, looks like errors were logged on crash but I can't be sure.
I have Ryzen 3 2200g CPU with Vega 8 graphics, is it strange that it's loading something CUDA? Isn't it a Nvidia thing?
```
~
❯ Applications/osu.AppImage
Failed to open VDPAU backend libvdpau_radeonsi.so: cannot open shared object file: No such file or directory
[AVHWDeviceContext @ 0x7f1ec0b164c0] VDPAU device creation on X11 display :0 failed.
[AVHWDeviceContext @ 0x7f1ec0ab44c0] Cannot load libcuda.so.1
[AVHWDeviceContext @ 0x7f1ec0ab44c0] Could not dynamically load CUDA
[AVHWDeviceContext @ 0x7f1ec0c27640] Cannot load libcuda.so.1
[AVHWDeviceContext @ 0x7f1ec0c27640] Could not dynamically load CUDA
osu!: malloc.c:4302: _int_malloc: Assertion `(unsigned long) (size) >= (unsigned long) (nb)' failed.
fish: Job 1, 'Applications/osu.AppImage' terminated by signal SIGABRT (Abort)

~ took 36m50s
❯```

**Reply by frenzibyte** [reply]
Looks like video hardware acceleration is not working well for you. Related: https://github.com/ppy/osu/discussions/16567. For now, disable video hardware acceleration in settings.

Also definitely don't reply in answered discussion threads, this is one that's completely irrelevant from your issue.

**Reply by bdach** [reply]
could also try installing a different FFmpeg version. either way out of scope of this discussion.
